pandas/src/algos_join_helper.pxi.in

```diff
@@ -0,0 +1,406 @@
+"""
+Template for each `dtype` helper function for join
+
+WARNING: DO NOT edit .pxi FILE directly, .pxi is generated from .pxi.in
```

Comment:
actually I see, maybe make sure this warning is in all of the .pxi.in files.....